recipe/bld.bat

```diff
@@ -1,9 +1,10 @@
 if not exist %PREFIX% mkdir %PREFIX%
 
 move lib\x64\* %LIBRARY_LIB%
-move include\* %LIBRARY_INC%
+if not exist %LIBRARY_INC%\CL mkdir %LIBRARY_INC%\CL
+move include\CL\* %LIBRARY_INC%\CL
 
- not exist %LIBRARY_PREFIX%\etc mkdir %LIBRARY_PREFIX%\etc
- not exist %LIBRARY_PREFIX%\etc\OpenCL mkdir %LIBRARY_PREFIX%\etc\OpenCL
- not exist %LIBRARY_PREFIX%\etc\OpenCL\vendors mkdir %LIBRARY_PREFIX%\etc\OpenCL\vendors
- type nul >> %LIBRARY_PREFIX%\etc\OpenCL\vendors\cuda.icd
+if not exist %LIBRARY_PREFIX%\etc mkdir %LIBRARY_PREFIX%\etc
+if not exist %LIBRARY_PREFIX%\etc\OpenCL mkdir %LIBRARY_PREFIX%\etc\OpenCL
+if not exist %LIBRARY_PREFIX%\etc\OpenCL\vendors mkdir %LIBRARY_PREFIX%\etc\OpenCL\vendors
```

Comment:
```suggestion
if not exist %LIBRARY_PREFIX%\etc\OpenCL\vendors mkdir %LIBRARY_PREFIX%\etc\OpenCL\vendors
```

Nit: Windows mkdir is recursive, so you only need one of these. https://stackoverflow.com/a/905237

Not a required change for merge, just pointing out.